macroslib/src/cpp/mod.rs

```diff
@@ -311,6 +311,7 @@ struct CppContext<'a> {
     conv_map: &'a mut TypeMap,
     target_pointer_width: usize,
     rust_code: &'a mut Vec<TokenStream>,
+    foreign_code_cache: &'a mut Vec<(SmolStr, String)>,
```

Comment:
Why not FxHashSet<(StringType, StringType)> ?

Reply:
Modified with a `FxHashSet<(SmolStr, String)>`.